enterprise/ha/src/test/java/jmx/HaBeanIT.java

```diff
@@ -68,7 +71,22 @@
     public static final ClusterRule clusterRule = new ClusterRule( HaBeanIT.class )
             .withInstanceSetting( setting( "jmx.port", STRING, (String) null ), intBase( 9912 ) )
             .withInstanceSetting( HaSettings.ha_server, stringWithIntBase( ":", 1136 ) )
-            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) );
+            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) )
+            .withStoreDirInitializer( new ClusterManager.StoreDirInitializer()
```

Comment:
I think it makes sense to have that behaviour, provided no tests depend on it, in which case it might be too complicated to get correct in time for the next patch release. Nevertheless, this is just test code, so it will not affect the quality of the production code, just the repeatability of the tests. I say go for it.